### DISCUSSION TITLE
Any way to disable tooltips on mouse hover?

### DISCUSSION THREAD

**Discussion by AlexAMCS** [valinet/ExplorerPatcher#3018] [Q&A] [answered]
It gets annoying after a while.

**Comment by pyrates999** [comment]
https://www.reddit.com/r/Windows11/comments/15p97r8/is_there_any_way_to_turn_off_these_annoying_text/

you should be googling first

**Reply by AlexAMCS** [reply]
That was the first thing I've done. It changed nothing, which is why I came here to ask for a solution specific to ExplorerPatcher, since I think the googled result only applies to the default taskbar.

**Reply by pyrates999** [reply]
Try this then that is for windows 10: https://winaero.com/disable-tooltips-windows-10/

**Reply by AlexAMCS** [reply]
No dice. Still getting tooltip popups on the taskbar, though it does disable the explorer popups.

**Reply by pyrates999** [reply]
Ok, keep googling then and post here if you find the answer then.

**Reply by AlexAMCS** [reply, marked as answer]
Windhawk with the "Disable Taskbar Thumbnails" now supports it, so it's not an issue anymore.

**Reply by Usv3re** [reply]
Umm that's not what I meant. 🤔

I mean just basic Windows hovering tooltips, like when you stop your mouse at the time and date in the tray another small window with date and time pops up.
![изображение](https://github.com/user-attachments/assets/573f131c-c627-4121-8840-c2649258cb41)

**Comment by Usv3re** [comment]
Would be great if it were included as a setting. It only bugs me in the system tray / taskbar.

**Comment by pyrates999** [comment]
try this: https://www.elevenforum.com/t/disable-tooltip-popups.10660/

**Reply by Usv3re** [reply]
Thanks, but it doesn't seem to work. Already googled it.

Either way, I'd prefer to remove those only from the taskbar and leave in the file explorer

**Reply by pyrates999** [reply]
what did you set the value to in the registry?
